### PR TITLE
Original change recommendation came from Castaldio86

### PR DESCRIPTION
**Pull request by shainw**
See related by @Castaldio86
https://github.com/Azure/Azure-Sentinel/pull/554

Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: Detections/SecurityEvent/RDP_RareConnection.yaml [hunk 1]
```diff
@@ -18,12 +18,12 @@ relevantTechniques:
...
 
-  let starttime = 1d;
-  let endtime = 14d;
+  let starttime = 14d;
+  let endtime = 1d;
   SecurityEvent
   | where TimeGenerated >= ago(endtime) 
   | where EventID == 4624 and LogonType == 10
-  | summarize StartTimeUtc = min(TimeGenerated), EndTimeUtc = max(TimeGenerated), ConnectinCount = count() 
+  | summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated), ConnectinCount = count()
```
Comment (ashwin-patil) [inline Detections/SecurityEvent/RDP_RareConnection.yaml:26]: Minor typo on variable - ConnectinCount

File: Detections/SecurityEvent/RDP_RareConnection.yaml [hunk 2]
```diff
@@ -32,6 +32,6 @@ query: |
   | where EventID == 4624
   | summarize by Computer = toupper(Computer), IpAddress, Account = tolower(Account)
   ) on Account, Computer
-  | summarize StartTimeUtc = min(StartTimeUtc), EndTimeUtc = max(EndTimeUtc), ConnectinCount = sum(ConnectinCount)  
+  | summarize StartTime = min(StartTime), EndTime = max(EndTime), ConnectinCount = sum(ConnectinCount)
```
Comment (ashwin-patil) [inline Detections/SecurityEvent/RDP_RareConnection.yaml:35]: Minor typo on variable - ConnectinCount

File: Detections/SecurityEvent/RDP_RareConnection.yaml (hunk 1 above)
Comment (shainw) [reply Detections/SecurityEvent/RDP_RareConnection.yaml]: done

File: Detections/SecurityEvent/RDP_RareConnection.yaml (hunk 2 above)
Comment (shainw) [reply Detections/SecurityEvent/RDP_RareConnection.yaml]: done

**Pull request merged**